app/controllers/provider_interface/notifications_controller.rb

```diff
@@ -1,13 +1,15 @@
 module ProviderInterface
   class NotificationsController < ProviderInterfaceController
     def update
-      current_provider_user.update!(
-        send_notifications: notification_params[:send_notifications],
-      )
-
       if FeatureFlag.active?(:configurable_provider_notifications)
-        current_provider_user.notification_preferences
```

Comment:
Should we be updating all preferences to true/false by default outside of the feature flag block to make sure we update the new values whilst being backward compatible until the feature flag is turned on?

Reply:
Yes, we should do that. We just need to make sure we run the migration #4269 before merging this PR.